lib/Game.js

```diff
@@ -21,14 +21,18 @@ class Game {
     const puzzleWordArray = this.currentAnswer.split(' ');
     this.answerLettersArray = puzzleWordArray.map((word) => {
       return word.split('');
-    })
+    });
   };
 
   checkDelimiters(character, positionNum) {
     if (character === '-') {
       $(`.letter-card-div-${positionNum}`).children().removeClass('hidden');
       $(`.letter-card-div-${positionNum}`).css({"background-color": "#27A067"});
     }
+    if (character === '\'') {
```

Comment:
This is a good conditional for the apostrophes.   We will need to try and test it to make sure it's working with the slash in there. If not maybe interpolation?

Reply:
Sounds good! Issue created regarding apostrophes, but likely priority P3 for now.